src/scan.h

```diff
@@ -45,6 +45,10 @@ hdl_kind_t source_kind(void);
 token_t processed_yylex(void);
 const char *token_str(token_t tok);
 
+void conda_id_init();
+void conda_id_exit();
+void conda_id_add(const char *ident, const char *value);
```

Comment:
I preferred the name before: "conda" doesn't really mean anything. How about `preprocessor_define_*`? E.g. `preprocessor_define_add()`. 

Also `void foo();` means `foo` takes an unknown number of arguments (i.e. `foo(1, 2, 3)` is allowed). Use `void foo(void);` instead if it takes no arguments (this is different to C++).

Reply:
I wanted something shorter, the previous one was way too long. True that `conda` does not mean anything.
WBU `pp_defines_init` and `pp_defines_add` ?

Reply:
Yes that's ok.